src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -441,9 +468,10 @@ protected IEnumerable<WorkloadId> WriteSDKInstallRecordsForVSWorkloads(IEnumerab
 
     internal static class InstallingWorkloadCommandParser
     {
-        public static readonly CliOption<string> WorkloadSetVersionOption = new("--version")
+        public static readonly CliOption<IEnumerable<string>> WorkloadSetVersionOption = new("--version")
         {
-            Description = Strings.WorkloadSetVersionOptionDescription
+            Description = Strings.WorkloadSetVersionOptionDescription,
```

Comment:
The help text you added is great, but I think the option text should also be updated to indicate it can be used in two ways.  Something like "A workload version to display, or a workload and its version joined by the '@' character.